hazelcast/src/main/java/com/hazelcast/client/console/ClientConsoleApp.java

```diff
@@ -395,9 +423,9 @@ public void run() {
             handleAtomicNumberGet(args);
         } else if ("a.set".equals(first)) {
             handleAtomicNumberSet(args);
-        } else if ("a.inc".equals(first)) {
+        } else if ("a.incrementAndGet".equals(first)) {
             handleAtomicNumberInc(args);
-        } else if ("a.dec".equals(first)) {
+        } else if ("a.decrementAndGet".equals(first)) {
```

Comment:
In my opinion keeping short forms would be better (we can still have long ones)

Reply:
There are also variants of these, such as `getAndDecrement` and `getAndIncrement`, I chose this format so that they would not confused with them.